blazor/chart/logarithmic-axis.md

```diff
@@ -175,7 +175,7 @@ The interval can be customized using the [Interval](https://help.syncfusion.com/
 
 ## Label format
 
-Using the [LabelFormat](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartCommonAxis.html#Syncfusion_Blazor_Charts_ChartCommonAxis_LabelFormat) property on an axis, it is possible to format the logarithmic labels to all globalize formats.
+Using the [LabelFormat](https://help.syncfusion.com/cr/blazor#Syncfusion_Blazor_Charts_ChartCommonAxis_LabelFormat) property on an axis, it is possible to format the logarithmic labels to all globalize formats.
```

Comment:
Can you recheck all CR links once and confirm with build team?

Reply:
updated proper link